gems/aws-sdk-core/spec/aws-sdk-core/profile/profile_file_factory_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require_relative '../../spec_helper'
+
+module AWS::SDK::Core
+  describe ProfileFileFactory do
+    subject { described_class }
+
+    context '#create' do
+      file = File.join(File.dirname(__FILE__),
+                       'config-file-location-tests.json')
+      test_cases = JSON.load_file(file)['tests']
+
+      # Even for Windows paths, ruby uses /
+      def normalize_path(path)
```

Comment:
Could create a `ProfileSpecHelper` module to contain all these testing utility methods? Similar to how we have `ProtocolTestsHelper` in V3

Reply:
This is the only place where it exists, I think it's fine just here in this spec.